packages/open-next/src/build/copyTracedFiles.ts

```diff
@@ -19,6 +19,23 @@ import { MIDDLEWARE_TRACE_FILE } from "./constant.js";
 
 const __dirname = url.fileURLToPath(new URL(".", import.meta.url));
 
+//TODO: we need to figure which packages we could safely remove
+const EXCLUDED_PACKAGES = [
+  "caniuse-lite",
+  "sharp",
+  // This seems to be only in Next 15
+  // Some of sharp deps are under the @img scope
+  "@img",
+];
+
+function isExcluded(srcPath: string) {
+  return EXCLUDED_PACKAGES.some((excluded) =>
+    new RegExp(`${path.sep}node_modules${path.sep}${excluded}${path.sep}`).test(
```

Comment:
- use `getCrossPlatformPathRegex`
- use `str.match(regexp)` instead of `.test` (which updates/uses `re.lastIndex` - it shouldn't cause issue here but it did for us in the cloudflare adapter so it would be better to avoid the construct)